councilmatic_core/management/commands/convert_attachment_text.py

```diff
@@ -41,7 +41,7 @@ def get_document_url(self):
         # Only apply this query to most recently updated (or created) bill documents.
         max_updated = BillDocument.objects.all().aggregate(max_updated_at=Max('bill__updated_at'))['max_updated_at']
```

Comment:
For next week: Running this command without the all documents flag results in nothing being converted, because we query for documents that were updated _after_ the max, which isn't possible. Also, since we're looking at the update timestamp on the bill, I'm not sure it's true that all bills from the same import have the same update timestamp. We may be better off doing something like converting everything updated in the last hour or two by default.

Reply:
the way the scraper importer will work is if a bill document is updated it, [will delete and recreate the bill document](https://github.com/opencivicdata/pupa/blob/master/pupa/importers/base.py#L360-L367).

So we don't need to check timestamps, we can just check to see if `full_text` is not set on `extras` and it should be.